### PR TITLE
[#224][mezzio-toggle] Update Mezzio package documentation

### LINKED ISSUE #224: [mezzio-toggle] Update Mezzio package documentation
Epic: #221 Mezzio Toggle Package
Package: https://github.com/pheature-flags/mezzio-toggle
Docs: https://pheatureflags.io/#/getting-started/laminas-and-mezzio-package

## Description:

Update the documentation regarding the Mezzio Toggle package with the following updates:
### Installation guide: 
* Explain how to install the package
* Explain the available drivers (inmemory, dbal)

### InMemory driver
* Explain what is and how to install
* Configuration
* Usage / Example

### Dbal driver
* Explain what is and how to install
* Configuration
* Usage / Example

### Rest API
* Explain what is and how to install
* Configuration
* Usage / Example

### Configuration section
* Display the full configuration reference and explain each property

Also, add minimal installation steps in the README package and add a link to the docs.

### PR DESCRIPTION
**Pull request by xserrat**
It closes #224

### CODE REVIEW

File: docs/getting-started/laminas-and-mezzio-package.md [hunk 1]
```diff
@@ -8,98 +8,462 @@ Feature toggle implementation out of the box for our [Laminas](https://getlamina
...
-// config/autoload/pheature-flags.php
+# config/config.php
+$aggregator = new ConfigAggregator([
+    // ...
+    \Pheature\Community\Mezzio\ToggleConfigProvider::class,
+    // ...
+```
 
-declare(strict_types=1);
+Once we have the bundle installed in our project, we need to choose one of the available storage drivers:
```
Comment (kpicaza) [inline docs/getting-started/laminas-and-mezzio-package.md]:
```suggestion
Once we have the package installed in our project, we need to choose one of the available storage drivers:
```

File: docs/getting-started/laminas-and-mezzio-package.md [hunk 2]
```diff
@@ -8,98 +8,462 @@ Feature toggle implementation out of the box for our [Laminas](https://getlamina
...
+);
+```
+
+> [pheature/dbal-toggle](/packages/dbal-toggle)
+
+#### Usage
+
+<span id="inmemory_driver_usage"></span>
+
+In this example, we are using a Mezzio Controller, assuming that we already have a Security layer to our controller (*).
```
Comment (kpicaza) [inline docs/getting-started/laminas-and-mezzio-package.md]:
```suggestion
In this example, we are using a PSR-15 Request Handler, assuming that we already have a Security layer to our controller (*). 
```

**Review by kpicaza — APPROVED** [review]
:clap::clap::clap: Good job!!! I added two small suggestions.

**Comment by xserrat** [comment]
Also, before merging this PR I'll create another one with some fixes to the mezzio-toggle that I found while I was writting this doc. I let you know to check it together!

File: docs/getting-started/laminas-and-mezzio-package.md (hunk 2 above)
Comment (xserrat) [reply docs/getting-started/laminas-and-mezzio-package.md]: Nice renaming 👏🏽 , done!!

File: docs/getting-started/laminas-and-mezzio-package.md (hunk 1 above)
Comment (xserrat) [reply docs/getting-started/laminas-and-mezzio-package.md]: Good catch! done!

**Pull request merged**